src/app/components/catalog/catalog.component.html

```diff
@@ -0,0 +1,15 @@
+<!--The content below is only a placeholder and can be replaced.-->
+
+<app-shopping-cart></app-shopping-cart>
```

Comment:
shopping cart must be in the header, not in the content part.
Please move It in app-root or create a header component and move it into the header component